manifests/pytorch-operator/base/deployment.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: pytorch-operator
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      name: pytorch-operator
+  template:
+    metadata:
+      labels:
+        name: pytorch-operator
+      annotations:
```

Comment:
Why do we disable istio for the PyTorchJob controller ?

Reply:
I remember there're failures somethese if sidecar is being used. Disabling istio is safer for jobs